bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -140,6 +144,49 @@ except KeyError:
     dec_chirp_dist = pycbc.pnutils.chirp_distance(dec_dist, mchirp)
     sorting = dec_chirp_dist.argsort()  # ascending order of dec chirp distance
 
+# Get the trigger SNRs and times
+# But only ones which are within a small window of the missed injection
+missed_inj_times = numpy.sort(f['injections/tc'][:][missed])
+
+# Note: Adding 2 * Earth diameter in light seconds to the window here
+# to allow for different IFO's arrival times of the injection
+safe_window = args.nearby_triggers_window + 0.085
+
+def nearby_missedinj(endtime, snr):
```

Comment:
should this function not have safe_window as an argument?

Reply:
Ah, well I guess it's written in a 'sloppy' way using global variables anyway (ie missed_inj_times and nearby_missed_inj).  So scratch that I guess